facebook_downloader/downloader.py

```diff
@@ -307,7 +307,21 @@ def get_account_ad_performance_for_single_day(ad_account: adaccount.AdAccount,
                 'level': 'ad',
                 'limit': 1000,
                 'time_range': {'since': single_date.strftime('%Y-%m-%d'),
-                               'until': single_date.strftime('%Y-%m-%d')}})
+                               'until': single_date.strftime('%Y-%m-%d')},
+                # By default only ACTIVE campaigns get considered.
+                'filtering': [{
+                    'field': 'ad.effective_status',
+                    'operator': 'IN',
+                    'value': ['ACTIVE',
+                              'PAUSED',
+                              'DELETED',
```

Comment:
Please remove `DELETED` status to avoid confusion. These ads will not be present regardless of the filter. 

>You can query insights on deleted objects if you have their IDs.
See: https://developers.facebook.com/docs/marketing-api/insights/v2.9